checkov/terraform/checks/data/aws/StarActionPolicyDocument.py

```diff
@@ -0,0 +1,28 @@
+from checkov.terraform.checks.data.base_check import BaseDataCheck
+from checkov.common.models.enums import CheckResult, CheckCategories
+
+
+class StarActionPolicyDocument(BaseDataCheck):
+    def __init__(self):
+        name = "Ensure IAM policies that allow \"*\" as the statement's actions are not created"
```

Comment:
suggested rephrase: Ensure IAM policies does not allow \"*\" as in statement's actions"
wdyt?